src/componets/request/test.jsx

```diff
@@ -0,0 +1,28 @@
+import React,{useState, useEffect} from "react";
+import axios from "axios";
+
+
+function Row(){
+	const [movies, setMovies] = useState([]);
+
+	useEffect(() =>{
+		async function fetchData(){
+			let fetchUrl ='https://ktc-ph-api.herokuapp.com/?format=json'
+			const request = await axios.get(fetchUrl);
+			setMovies(request);
+			console.log(movies)
+			return request;
+		}
+		fetchData();
```

Comment:
Try adding catch. explore adding timeout as well.

Reply:
done na po.. for timeout future setup ko po